app/script/view_model/list/TemporaryGuestViewModel.js

```diff
@@ -36,7 +36,7 @@ z.viewModel.list.TemporaryGuestViewModel = class TemporaryGuestViewModel {
     this.userRepository = repositories.user;
     this.callingRepository = repositories.calling;
     this.multitasking = mainViewModel.content.multitasking;
-    this.mediaRepository = repositories.media;
+    this.permissionRepository = repositories.media;
```

Comment:
Humm there seem to be something weird here `permission... = ...media`

Reply:
Thanks for the catch